src/sphinx_ansible_theme/extrabody.html

```diff
@@ -17,7 +17,7 @@
 </div>
 {%- endif %}
 
-<a class="DocSite-nav" href="/" style="padding-bottom: 30px;">
+<a class="DocSite-nav" href="{{ documentation_home_url | default('/', true) }}" style="padding-bottom: 30px;">
```

Comment:
This just rely on the default value from `theme.conf` + theme vars are prefixed, unlike HTML context:
```suggestion
<a class="DocSite-nav" href="{{ theme_documentation_home_url }}" style="padding-bottom: 30px;">
```

Reply:
Implemented in 79ebde21fe6279a39f489c954865033f85a00089.